src/ex_cmds2.c

```diff
@@ -460,26 +460,33 @@ ex_listdo(exarg_T *eap)
 
     if (curwin->w_p_wfb)
```

Comment:
Currently 'wfb' is checked for by `:windo` and such, where being able to switch buffers may not be 100% necessary. Don't have much of an opinion on the matter, though.